core/src/main/java/io/questdb/std/SwarUtils.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.std;
+
+public final class SwarUtils {
+
+    private SwarUtils() {
+    }
+
+    /**
+     * Broadcasts the given byte to a long.
+     */
+    public static long broadcast(byte b) {
+        return 0x101010101010101L * (b & 0xffL);
+    }
+
+    /**
+     * Returns non-zero result in case if the input contains a zero byte.
+     * <p>
+     * Each zero byte of the input is replaced with 0x80 in the output.
```

Comment:
You can also use `{@value #ZERO_CHECK_BYTE}` here, which will inline the value in the Javadoc output.